### PR TITLE
feat(paramsAndFixes): param units, descriptions and Vuex state alterations

### PR DESCRIPTION
**Pull request by Matt-Hayter**

### CODE REVIEW

**Comment by TJ-io** [comment]
PR name too long

File: frontend/src/components/ConfigBar/ConfigBar.vue [hunk 1]
```diff
@@ -84,10 +82,17 @@ export default {
     },
     //Update tabsData[i].isActive props if tab changes
     onTabOneClick() {
-      this.$emit("tabOneActive")
+      //Close all parameter descriptions for current tab
+      for (let i = 0; i < this.$refs.item.length; i++) {
+        this.$refs.item[i].tabClick()
+      }
```
Comment (TJ-io) [inline frontend/src/components/ConfigBar/ConfigBar.vue]:
```suggestion
      this.$refs.item.forEach((item) => item.tabClick())
      
      // OR
      
      this.$refs.item.forEach((item, i) => this.$refs.item[i].tabClick())
```

You should try to use ES6 arrow functions where you can, more clean and generally the standard. (One of the above will work, ideally the top but bottom may only work)

File: frontend/src/components/ConfigBar/ConfigBar.vue [hunk 2]
```diff
@@ -84,10 +82,17 @@ export default {
...
+      setTimeout(() => this.$emit("tabOneActive"), 15)
     },
     onTabTwoClick() {
-      this.$emit("tabTwoActive")
+      for (let i = 0; i < this.$refs.item.length; i++) {
+        this.$refs.item[i].tabClick()
+      }
```
Comment (TJ-io) [inline frontend/src/components/ConfigBar/ConfigBar.vue]: convert to ES6 Arrow func

File: frontend/src/components/ConfigBar/components/PresetButtons.vue [hunk 3]
```diff
@@ -115,14 +114,17 @@ export default {
     activeUser() {
       return this.$store.state.activeUser;
     },
+    simRunning() {
+      return this.$store.state.simRunning
```
Comment (TJ-io) [inline frontend/src/components/ConfigBar/components/PresetButtons.vue:118]: Some lines you have `;` some you don't, make sure you are consistent - seems you aren't running code formatter

File: frontend/src/components/ConfigBar/components/PresetButtons.vue [hunk 4]
```diff
@@ -156,7 +158,7 @@ export default {
     },
     onClickSave() {
       //So button only toggles preset naming modal when signed in
-      if (this.$store.state.activeUser.isActive == true) {
+      if (this.activeUser.isActive == true) {
```
Comment (TJ-io) [inline frontend/src/components/ConfigBar/components/PresetButtons.vue]:
```suggestion
      if (this.activeUser.isActive) {
```

File: frontend/src/views/CompetingSpecies/CompetingSpecies.vue [hunk 5]
```diff
@@ -357,11 +371,11 @@ export default {
       try {
         const path = "http://localhost:5000/CompetingSpecies/AllPresets";
         const payload = {
-          userEmail: this.$store.state.activeUser.email
+          userEmail: this.activeUser.email
```
Comment (TJ-io) [inline frontend/src/views/CompetingSpecies/CompetingSpecies.vue:374]:
```suggestion
          userEmail: this.activeUser.email,
```

consistency with commas at EOL object entries (formatter will sort)

**Review by TJ-io — CHANGES_REQUESTED** [review]
couple changes - also shorten the PR name

File: frontend/src/components/ConfigBar/ConfigBar.vue (hunk 1 above)
Comment (TJ-io) [reply frontend/src/components/ConfigBar/ConfigBar.vue]: if this.$refs.item is an array []

File: frontend/src/components/ConfigBar/components/PresetButtons.vue (hunk 3 above)
Comment (Matt-Hayter) [reply frontend/src/components/ConfigBar/components/PresetButtons.vue:118]: I'll do this once merged to main, to avoid conflicts

**Review by TJ-io — APPROVED** [review]
LGTM!

**Pull request merged**